### PR TITLE
Importing and exporting on ckan 2.4

### PR DESCRIPTION
**Pull request by vitorbaptista**
This is now ready for review. The failing test is because of a problem in CKAN fixed on https://github.com/ckan/ckan/pull/2801.

/cc @pwalsh @amercader

### CODE REVIEW

**Comment by vitorbaptista** [comment]
Actually, the current error is because `tabulator` isn't on PyPI (https://github.com/okfn/tabulator-py/issues/23).

File: README.markdown [hunk 1]
```diff
@@ -1,181 +1,71 @@
...
-2. Make sure Ubuntu's package index is up to date:
+1. If you're using a virtual environment for CKAN, activate it doing, for example:
 
-        sudo apt-get update
+        source /usr/lib/ckan/default/bin/activate
+    
+2. Install the `ckanext-datapackager` extension using pip:
 
-3. Install the Ubuntu packages that Data Packager requires:
+        pip install ckanext-datapackager
```
Comment (amercader) [inline README.markdown]: This won't install requirements, perhaps is better to git clone + python setup.py develop + pip install -r requirements.txt

**Comment by amercader** [comment]
This is looking really good @vitorbaptista 

Some quick questions after a first quick look:

* Where has the Additional info gone? http://i.imgur.com/5wmJI58.png

* I don't seem to be able to update an existing dataset that was imported from a data package reuploading the same datapackage. Is this intended? Or do I need to provide an `id` on the data package?

* When uploading a zip, if the datapackage.json is not at the root level (eg downloading a zip from a GitHub repo gives you a folder) you get this confusing error: http://i.imgur.com/5aoEMS3.png. Would it be difficult to provide support for the zipped folder?

File: README.markdown (hunk 1 above)
Comment (vitorbaptista) [inline README.markdown]: Might be better to fix `setup.py` so `pip install ckanext-datapackager` installs its dependencies. Cloning the repository makes it harder for the users to use a specific version of the extension, and not simply following master.

**Comment by vitorbaptista** [comment]
1. Fixed on https://github.com/ckan/ckanext-datapackager/commit/1222658d04fc98ee4a773acdd6bfc98581ec4cb4
2. That isn't implemented;
3. These issues are with https://github.com/okfn/datapackage-py. I created an issue to improve the exception messages (https://github.com/okfn/datapackage-py/issues/27), and another to decide if we should support loading ZIP files with datapackage.json not in the root level (https://github.com/okfn/datapackage-py/issues/29)

**Pull request merged**